spec/policies/participant_policy_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ParticipantPolicy, type: :policy do
+  subject { described_class.new(acting_user, user_under_test) }
+
+  %i[early_career_teacher_profile mentor_profile].each do |profile_type|
+    context "when the participant has #{profile_type}" do
+      # rubocop:disable Rails/SaveBang
+      let(:user_under_test) { create(profile_type).user }
+      # rubocop:enable Rails/SaveBang
```

Comment:
Why do we need this rubocop stuff here?

Reply:
Eugh. So, it thinks this is an ActiveRecord#create, and wants me to use create!. It's because I'm not passing a symbol